astropy/table/pprint.py

```diff
@@ -392,7 +392,8 @@ def _pformat_col_iter(self, col, max_lines, show_name, show_unit, outs,
         if multidims:
             multidim0 = tuple(0 for n in multidims)
             multidim1 = tuple(n - 1 for n in multidims)
-            trivial_multidims = np.prod(multidims) == 1
+            multidims_all_ones = np.prod(multidims) == 1
```

Comment:
I'd perhaps define `multidim_size = np.prod(multidims)` and compare that with 1 and 0 below, but really this is very clear too.